target/linux/ath79/dts/ar7161_aruba_ap-175.dts

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar7100.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "aruba,ap-175", "qca,ar7161";
+	model = "Aruba AP-175";
+
+	chosen {
+		bootargs = "console=ttyS0,9600";
+	};
+
+	aliases {
+		led-boot = &led_power_red;
+		led-failsafe = &led_power_red;
+		led-upgrade = &led_power_red;
+		label-mac-device = &eth0;
+	};
+
+	extosc: ref {
+		compatible = "fixed-clock";
```

Comment:
Please remove this node. it has been moved to SoC dtsi. https://github.com/openwrt/openwrt/commit/520c90854ca73eb6c3d8feeda59766c90bdd4144

Reply:
Done with 3f1e26b2941c4500aa1d8153bc641e8dbff7c7d4.